immuni_analytics/helpers/request.py

```diff
@@ -35,6 +36,12 @@ class ServerUnavailableError(ImmuniException):
     """
 
 
+class TooManyRequestsError(ImmuniException):
```

Comment:
Is this happening because we are calling several times with the same device id? If you know

Reply:
Nope, I don't know right now, and the ones I saw are 400. Yet, I think it's reasonable to retry on 429. Ideally, we should add some circuit-breaker logic, but I don't think it's worth right now.

Reply:
OK then